static-instrumenter/agent-instrumenter/build.gradle.kts

```diff
@@ -3,7 +3,6 @@ import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
 plugins {
   id("otel.java-conventions")
   id("com.github.johnrengelman.shadow") version "7.1.2"
```

Comment:
By the way, we should never have version numbers in plugins within artifacts as they all need to be the same in the build (this could potentially catch accidentally introducing an unused plugin like test-sets). Shadow in particular is already there

https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/settings.gradle.kts#L4

Reply:
So basically this is a point we need to add to CONTRIBUTING.md

https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/CONTRIBUTING.md

Reply:
I'll send PR 👍